src/idsk/components/header-extended/header-extended.js

```diff
@@ -59,7 +59,10 @@ HeaderExtended.prototype.init = function () {
     var $hamburgerMenuButton = $module.querySelector('.idsk-js-header-extended-side-menu');
     var $closeMenuButton = $module.querySelector('.idsk-header-extended__mobile-close');
     if ($hamburgerMenuButton && $closeMenuButton) {
-        $hamburgerMenuButton.addEventListener('click', this.showMobilMenu.bind(this));
+        $hamburgerMenuButton.addEventListener('click', function(){
+            this.showMobilMenu();
+            if(document.activeElement == $hamburgerMenuButton) $lastMenuElement.focus();
```

Comment:
pouzivajme stale kucerave zatvorky pri podmienkach aj pri takych ako tu jednoriadkovych, ver mi obcas to sposobi nemale problemy :) plus mozno ta podmienka by mohla byt rovno sucastou funkcie showMobilMenu ci?